tests/tests_app/cli/test_connect.py

```diff
@@ -20,16 +20,10 @@
 
 def monkeypatch_connection(monkeypatch, tmpdir, ppid):
     connection_path = os.path.join(tmpdir, ppid)
-    try:
-        monkeypatch.setattr("lightning_app.cli.commands.connection._clean_lightning_connection", MagicMock())
-        monkeypatch.setattr("lightning_app.cli.commands.connection._PPID", ppid)
-        monkeypatch.setattr("lightning_app.cli.commands.connection._LIGHTNING_CONNECTION", tmpdir)
-        monkeypatch.setattr("lightning_app.cli.commands.connection._LIGHTNING_CONNECTION_FOLDER", connection_path)
-    except ModuleNotFoundError:
-        monkeypatch.setattr("lightning.app.cli.commands.connection._clean_lightning_connection", MagicMock())
-        monkeypatch.setattr("lightning_app.cli.commands.connection._PPID", ppid)
-        monkeypatch.setattr("lightning.app.cli.commands.connection._LIGHTNING_CONNECTION", tmpdir)
-        monkeypatch.setattr("lightning.app.cli.commands.connection._LIGHTNING_CONNECTION_FOLDER", connection_path)
+    monkeypatch.setattr("lightning_app.cli.commands.connection._clean_lightning_connection", MagicMock())
```

Comment:
why is it lightning_app here insetad of lightning.app?

Reply:
it is handled (it was even before) by code/test adustmensts in CI